game/consumers.py

```diff
@@ -48,3 +51,49 @@ def move_token(self, id, x, y):
 
     async def token_position_changed(self, event):
         await self.send(text_data=json.dumps(event["data"]))
+
+
+class ChatConsumer(WebsocketConsumer):
+    def connect(self):
+        self.room_name = self.scope['url_route']['kwargs']['room_name']
```

Comment:
Double quotes are preferable to single quotes

Reply:
done